src/software/networking/proto_udp_client.h

```diff
@@ -64,7 +66,8 @@ class ProtoUdpClient
     // Buffer to hold serialized protobuf data
     std::string data_buffer;
 
-    static constexpr unsigned int MAX_BUFFER_LENGTH = 9000;
+    static constexpr unsigned int MAX_BUFFER_LENGTH      = 9000;
+    static constexpr unsigned int UDP_RECEIVE_TIMEOUT_MS = 100;
```

Comment:
this timeout value wasn't enough for me... none of the messages went through. 500ms worked, haven't tried smaller though

Reply:
are u sure the interfaces were correct? i had that happen to me and it was because i had the --interface not match properly from "field testing" on sunday

I think the increment by 2 issue could be because you clicked it twice during the 1/2 a second delay?

Reply:
the interface have to be correct because it worked when I increased the timeout
I didn't double click

Reply:
I didn't see the problem with double incrementing

Reply:
I can't repro the double incrementing issue locally either but I can repro the delay, so for the sake of stability i've brought back the old libraries that work well and will leave it untouched until after competition